packages/botonic-cli/src/util/bot-config.ts

```diff
@@ -46,6 +63,49 @@ export class BotConfig {
         botonic_cli_version: botonicCliVersion,
       },
       packages,
+      tools: configLoaded.tools,
+      payloads: configLoaded.payloads,
+      webviews: configLoaded.webviews,
+    }
+  }
+  static async getTools(appDirectory: string) {
+    try {
+      const botConfigPath = path.join(appDirectory, '.src/bot-config.ts')
+      const output = await this.getOutputByCommand(
+        `npx ts-node ${botConfigPath}`
+      )
+      console.log('output', output)
+    } catch (err: any) {
+      console.error(`Error: ${err.message}`)
+    }
+    return []
+  }
+
+  static async loadBotConfig(appDirectory: string) {
+    try {
+      // Register ts-node to load TypeScript in real time
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      require('ts-node').register({
+        transpileOnly: true,
+        compilerOptions: {
+          module: 'commonjs',
+          target: 'es2018',
+          esModuleInterop: true,
+          allowSyntheticDefaultImports: true,
+        },
+      })
+
+      const configPath = path.resolve(appDirectory, 'src/bot-config.ts')
+
+      // Clean cache to reload if necessary
+      delete require.cache[require.resolve(configPath)]
+
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const { botConfig } = require(configPath)
+      return botConfig
+    } catch (error) {
+      console.error('Error cargando bot config:', error)
+      throw error
```

Comment:
Doubt: Shouldn't we check first if the path exists and if it doesn't exist don't do nothing instead of throwing an error if the file doesn't exist? The bot may not have this file and the deploy shouldn't fail if it doesn't exist.